spec/lib/kie/service_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe Kie::Service do
+  let(:kie_ex) { KieClient::ApiError.new("kie_error") }
+  let(:options) { {} }
+
+  describe '.call' do
+    it 'raises a KieClient::ApiError' do
+      expect do
+        described_class.call(KieClient::ProcessInstancesBPMApi, options) do |_api|
+          raise kie_ex
+        end
+      end.to raise_exception(Exceptions::KieError)
+    end
```

Comment:
I think we might want another spec here testing this line https://github.com/RedHatInsights/approval-api/blob/master/lib/kie/service.rb#L24.

We should be able to do something like this, I think?
```
it "sets the authorization headers" do
  auth_headers = described_class.call(KieClient::ProcessInstancesBPMApi, options) do |api|
    api.api_client.default_headers["Authorization"]
  end

  expect(auth_headers).to eq("<Whatever the basic auth token is>")
end
```

I'm just thinking that if that line changes, the current spec doesn't really test anything about it and other than setting up the `KieClient` configuration, that seems to be an important step that we're not covering.